deploy.sh

```diff
@@ -23,7 +23,7 @@ fi
 
 msg "Removing the old website"
 pushd public
-git rm -rf *
+git rm -rf public/*
```

Comment:
The line above this actually changes the directory to `public/`, see [how do I use `pushd` and `popd` commands](http://unix.stackexchange.com/questions/77077/how-do-i-use-pushd-and-popd-commands), so the change you made is not correct.

However, these three lines can probably be replaced with `rm -rf public/`; I don't recall if there was a reason to do it the way it is now. If you want to make that change, I'd happily merge it :)